app/src/main/res/xml/preferences_playback.xml

```diff
@@ -78,6 +66,67 @@
                 android:title="@string/pref_playback_time_respects_speed_title"/>
     </PreferenceCategory>
 
+    <PreferenceCategory android:title="@string/hardware_control">
+        <ListPreference
+            android:entryValues="@array/hardware_forward_single_tap_action_values"
+            android:entries="@array/hardware_forward_single_tap_action_options"
+            android:title="@string/pref_hardwareForwardSingleTapAction_title"
+            android:key="prefHardwareForwardAction0"
+            android:summary="@string/pref_hardwareForwardSingleTapAction_sum"
+            android:defaultValue="SEEK_FORWARD"
+            app:useStockLayout="true"/>
+        <ListPreference
+            android:entryValues="@array/hardware_back_single_tap_action_values"
+            android:entries="@array/hardware_back_single_tap_action_options"
+            android:title="@string/pref_hardwareBackSingleTapAction_title"
+            android:key="prefHardwareBackAction0"
+            android:summary="@string/pref_hardwareBackSingleTapAction_sum"
+            android:defaultValue="SEEK_BACKWARD"
+            app:useStockLayout="true"/>
+        <SwitchPreference
+            android:defaultValue="false"
+            android:enabled="true"
+            android:key="prefAdvancedHardwareActions"
+            android:summary="@string/pref_advancedHardwareActions_sum"
+            android:title="@string/pref_advancedHardwareActions_title"/>
```

Comment:
I wanted to present a warning to the user before they enable multi-tap actions, because many hardware devices will have their own custom behaviors and prevent these preferences from working. This implements it by a switch with a 'Warning' in the summary.

A dialog box or separate screen might be better, but a little more complicated to implement based on my limited understanding of Android preferences. I can look into this more if desired.